tests/tests_deps.yml

```diff
@@ -26,7 +26,7 @@
 
     - assert:
         that:
-          - "{{ blivet_output.packages }} == ['lvm2', 'xfsprogs']"
+          - "{{ blivet_output.packages|to_json }} == ['lvm2', 'xfsprogs']"
```

Comment:
Does this work?
```yaml
           - blivet_output.packages == ['lvm2', 'xfsprogs']
```
?
If it doesn't work, is it because `blivet_output.packages` is not sorted?  So then, does this work?
```yaml
           - blivet_output.packages|sort == ['lvm2', 'xfsprogs']
```

Reply:
> Does this work?
> 
> ```yaml
>            - blivet_output.packages == ['lvm2', 'xfsprogs']
> ```
> 
Yeah, this format works well.


> ?
> If it doesn't work, is it because `blivet_output.packages` is not sorted? So then, does this work?
> 
> ```yaml
>            - blivet_output.packages|sort == ['lvm2', 'xfsprogs']
> ```

Reply:
For `when` statements and `assert.that` statements, you generally don't need quoting and `{{ }}` expansion

Reply:
OK, if you agree, I will cook one patch to remove all the quoting and `{{ }}` expansion in `when` statements and `assert.that` statements

Reply:
@yizhanglinux thanks, but please start with updating this PR to remove it at this problematic location, you don't need to fix it everywhere immeadiately.

Reply:
> @yizhanglinux thanks, but please start with updating this PR to remove it at this problematic location, you don't need to fix it everywhere immeadiately.

OK, will update it later.